integration/setup/common.go

```diff
@@ -104,123 +117,57 @@ func SkipForPostgresWithReason(tb testing.TB, reason string) {
 	}
 }
 
-// checkMongoDBURI returns true if given MongoDB URI is working.
-func checkMongoDBURI(tb testing.TB, ctx context.Context, uri string) bool {
```

Comment:
This is essentially the same func as `setupClient(...` 
https://github.com/FerretDB/FerretDB/pull/1857/files#diff-e44bf8c258ee6bba05163a94822b0605ba181d26efa013179df8087fa3366e5dR261

We don't need to create client in `checkMongoDBURI` then create another client in `setupClient` in every  compat test.